docs/InstallGuide.md

```diff
@@ -131,294 +138,221 @@ To automatically launch the mysql server, execute the following command.
-    BOOTPROTO=none
-    ONBOOT=yes
-    HOTPLUG=no
-    #
-    # vi /etc/sysconfig/network-scripts/ifcfg-br0
-    DEVICE=br0
-    DEVICETYPE=ovs
-    TYPE=OVSBridge
-    ONBOOT=yes
-    BOOTPROTO=static
-    IPADDR=172.16.20.41
-    NETMASK=255.255.255.0
-    HOTPLUG=no
-    OVS_EXTRA="
-     set bridge     ${DEVICE} protocols=OpenFlow10,OpenFlow12,OpenFlow13 --
-     set bridge     ${DEVICE} other_config:disable-in-band=true --
-     set bridge     ${DEVICE} other-config:datapath-id=00004e6d2b508f4c --
-     set bridge     ${DEVICE} other-config:hwaddr=02:01:00:00:00:01 --
-     set-fail-mode  ${DEVICE} standalone --
-     set-controller ${DEVICE} tcp:127.0.0.1:6633
-    "
+```
+# initctl start vnet-vnmgr
+# initctl start vnet-webapi
+```
 
-### OpenVNet package installation
+Subsequent database records are required for the sake of the sample environment.
 
-Please install OpenVNet according to the "Pre-setup" and "OpenVNet installation" sections of "Preliminary Operations and Installation".
+#### Datapath
 
-The config files (vnmgr.conf, vna.conf and webapi.conf) in the /etc/openvnet directory do not need to be edited. They're set up for a single node installation by default.
+```
+# ./bin/vnctl datapaths add --uuid dp-test1 --display-name test1 --dpid 0x0000aaaaaaaaaaaa --node-id vna
+```
 
-### Other package installation
+* dpid
 
-    # yum install qemu-kvm
-    # yum install tcpdump
+The datapath ID specified in `/etc/sysconfig/network-scripts/ifcfg-br0`
 
+* node-id
 
-Start the required processes for OpenVNet
------------------------------------------
+The ID of the vna written in `/etc/openvnet/vna.conf`
 
-### redis
 
-    # chkconfig --list redis
-    redis           0:off   1:off   2:off   3:off   4:off   5:off   6:off
-    # chkconfig redis on
-    # chkconfig --list redis
-    redis           0:off   1:off   2:on    3:on    4:on    5:on    6:off
-    # service redis start
+#### Network
 
+```
+# ./bin/vnctl networks add --uuid nw-test1 --display-name testnet1 --ipv4-network 10.100.0.0 --ipv4-prefix 24 --network-mode virtual
+```
 
-### mysqld
+* ipv4-network
 
-    # chkconfig --list mysqld
-    mysqld          0:off   1:off   2:off   3:off   4:off   5:off   6:off
-    # chkconfig mysqld on
-    mysqld          0:off   1:off   2:on    3:on    4:on    5:on    6:off
-    # service mysqld start
+The IPv4 network address.
 
+* ipv4-prefix
 
-Start OpenVNet
----------------
-    # reboot
-    #
-    # initctl start vnet-vnmgr
-    # initctl start vnet-webapi
-    # initctl start vnet-vna
-    # initctl list | grep vnet # <- This line confirms that all three processes have started successfully.
-    vnet-vna start/running, process 2505
-    vnet-vnmgr start/running, process 2286
-    vnet-webapi start/running, process 2296
+The IPv4 network prefix. (default 24)
 
+* network-mode
 
-Download test environment
---------------------------
+The mode of the network to create. Use `virtual` in case of creating virtual network.
```

Comment:
I think at this point people might get confused and follow along without really understanding why what they are creating is a virtual network. I would say something like: "We are currently creating the virtual network 10.100.0.0/24 mentioned at the top of this guide. That's why we have to set the mode as virtual."

Also we should explain somewhere what the difference is between network mode virtual and physical but I think this is not the right place for that. Perhaps in the jargon dictionary?